### PR TITLE
Storyshot integration

### PR DESCRIPTION
**Pull request by oahmedqureshi**

### CODE REVIEW

File: client/.storybook/.babelrc [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+	"presets": ["es2015", "react"],
```
Comment (mashhoodr) [inline client/.storybook/.babelrc]: @oahmedqureshi looks like an indentation issue

File: client/src/stories/EMNavbar.js [hunk 2]
```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import { storiesOf } from '@storybook/react';
 import { EMNavbar } from '../components/EMNavbar';
 import { BrowserRouter as Router } from 'react-router-dom';
+
```
Comment (mashhoodr) [inline client/src/stories/EMNavbar.js]: @oahmedqureshi please dont commit white spaces

File: client/.storybook/.babelrc (hunk 1 above)
Comment (oahmedqureshi) [reply client/.storybook/.babelrc]: I'm not committing these extra spaces

**Comment by oahmedqureshi** [comment]
I haven't seen anything like generating separate snapshot for each story.

**Comment by mashhoodr** [comment]
@oahmedqureshi please check ST frontend, i saw they had several files.

**Comment by mashhoodr** [comment]
@oahmedqureshi following up on the multiple snapshot files, can you please update on that

**Comment by oahmedqureshi** [comment]
Yes ST is generating separate snapshots.snap file for each test. We are still trying to figure out how they configure it.

**Comment by mashhoodr** [comment]
Ok please create a new PR with a fix for that, Im merging this one for now.

**Pull request merged**